metadata.json

```diff
@@ -132,6 +144,14 @@
       "name": "simp/tlog",
       "version_requirement": ">= 0.1.0 < 1.0.0"
     },
+    {
+      "name": "simp/tuned",
+      "version_requirement": ">= 0.1.0 < 1.0.0"
+    },
+    {
+      "name": "simp/useradd",
+      "version_requirement": ">= 0.2.2 < 1.0.0"
+    },
     {
```

Comment:
Missing simp/upstart ?  Used by simp::mountpoints and sim::ctrl_alt_del.  simp::mountpoints is in the list of classes in data-in-modules.